src/core/core/rpc_engine.cpp

```diff
@@ -144,7 +144,12 @@ namespace dsn {
                 req->server_address.group_address()->leader_forward();
             }
 
-            call->set_delay(delay_ms);
+            // the "delay_ms" is simulator's net_delay_milliseconds,
+            // the "call->delay_milliseconds()" is user specified delay time to dispatch response task,
```

Comment:
I would remain the current situation unchanged, and add this later when there is a scenario requiring the sum of the latency.